### PR TITLE
update proto

### PR DESCRIPTION
**Pull request by 2yuri**

### CODE REVIEW

**Review by samufacanha2 — APPROVED** [review]

File: lib/transaction/proto/contracts.proto [hunk 1]
```diff
@@ -9,13 +9,16 @@ message TransferContract {
 	bytes ToAddress    = 1;
 	bytes AssetID      = 2;
 	int64 Amount       = 3;
+	int64 KDARoyalties = 4;
+  int64 KLVRoyalties = 5;
```
Comment (fbsobreira) [inline lib/transaction/proto/contracts.proto:13]: check indent

File: lib/transaction/proto/contracts.proto [hunk 2]
```diff
@@ -102,16 +131,23 @@ message AssetTriggerContract {
...
-    bytes              ToAddress    = 3;
-	  int64              Amount       = 4;
-	  bytes              MIME         = 5;
-	  string             Logo         = 6;
-	  map<string,string> URIs         = 7;
-    RolesInfo          Role         = 8;
-    StakingInfo        Staking      = 9;
+    EnumTriggerType    TriggerType  = 1 ;
+	bytes              AssetID      = 2 ;
+    bytes              ToAddress    = 3 ;
```
Comment (fbsobreira) [inline lib/transaction/proto/contracts.proto:141]: update indent

File: .DS_Store
Comment (fbsobreira) [inline .DS_Store:1]: remove .DS_Store

**Review by fbsobreira — CHANGES_REQUESTED** [review]
small changes...

**Review by fbsobreira — APPROVED** [review]

**Pull request merged**